vendor/darwin/Foundation/NSScreen.odin

```diff
@@ -1,5 +1,16 @@
 package objc_Foundation
 
-@(objc_class="NSScreen")
-Screen :: struct {using _: Object}
-// TODO: implement NSScreen
+@(objc_class = "NSScreen")
+Screen :: struct {
+	using _: Object,
+}
+
+@(objc_type = Screen, objc_name = "main")
+Screen_main :: proc "c" () -> ^Screen {
+	return msgSend(^Screen, Screen, "main")
+}
+
+@(objc_type = Screen, objc_name = "frame")
+Screen_frame :: proc "c" (self: ^Screen) -> ^Rect {
+	return msgSend(^Rect, self, "main")
```

Comment:
This binding is wrong, the msg name would be something like "frame". It also doesn't return a pointer just a plain old Rect.